src/Dashboard/Propertypost.jsx

```diff
@@ -0,0 +1,334 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import { useParams } from "react-router-dom";
+import { useDispatch } from "react-redux";
+// import "./"
+import axios from "axios";
+import Selectoption from "./Selectoption";
+// import { addProperty, propertyError } from "../slice/propertyslice";
+import "./propertyProp.css";
+
+const url = "http://localhost:3006/v1/agent/properties";
+
+const Propertypost = () => {
+
+  const getToken = JSON.parse(localStorage.getItem('data'));
+
+  let config = {
+    "headers": {
+      'Authentication': getToken
+    }
+  }
+console.log(config, getToken)
+  const dispatch = useDispatch();
+  const defaultFormData = {
+    title: "",
+    address: "",
+    purpose: "",
+    propertyType: "",
+    city: "",
+    image: "",
+    noOfBed: "",
+    noOfBath: "",
+    garages: "",
+    landSize: "",
+    locality: "",
+    Neighbourhood: "",
+    description: "",
+  };
+  const [formData, setFormData] = useState(defaultFormData);
+
+  const handleChange = (e) => {
+    const { value, name } = e.target;
+    setFormData({ ...formData, [name]: value });
+  };
+
+  console.log(formData)
+
+  const handleSubmit = async (e) => {
```

Comment:
You need some kind of loading state or preloader to let the user know their data is being submitted, I don't see that implementation here.